### PR TITLE
fix: no-wildcard check

### PR DESCRIPTION
**Pull request by rickardl**
# Pull Request

## Description

This PR fixed a bug where it tries to add a wildcard even though you specifically don't request it

### Type of change

Please delete options that are not relevant.

- [X] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

## How Has This Been Tested?

Locally tested against my terraform plan

### Checklist

- [X] `terraform fmt` and `terraform validate` both work from the root and `examples/` directories (look in CI for an example)
- [X] Provide an updated example that utilizes newly created resources, or for more advanced additions a new example under the examples directory.
- [ ] Example plan output in this PR (in lieu of CI)
- [ ] Docs have been added/updated (for bug fixes/features)
- [X] Any breaking changes are noted in the description above
- [ ] I have made corresponding changes to the documentation
- [X] My changes generate no new warnings
- [X] My code follows the style guidelines of this project
- [X] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

File: main.tf [hunk 1]
```diff
@@ -26,7 +26,7 @@ resource "aws_route53_record" "cert_validation" {
 }
 
 resource "aws_acm_certificate_validation" "main" {
-  count           = "${var.wait_for_validation == "true" ? 1 : 0}"
+  count           = "${2var.wait_for_validation == "true" ? 1 : 0}"
```
Comment (itsdalmo) [inline main.tf]: 2var must be a typo?

Comment (rickardl) [reply main.tf]: LOL!

Comment (rickardl) [reply main.tf]: @itsdalmo thanks for spotting

**Review by itsdalmo — APPROVED** [review]
LGTM 😂

**Pull request merged**